.github/workflows/ci_python.yml

```diff
@@ -105,15 +105,25 @@ jobs:
             dnf -y install clang protobuf-devel lld
             clang --version
             protoc --version
+      - name: Determine Version
+        if: github.event_name != 'release'
+        run: |
+          pip install tomlkit
+          VERSION=$(python ../scripts/get_version.py pyproject.toml project.version)
+          WHEEL=dist/kaskada-${VERSION}-cp38-abi3-manylinux_2_28_x86_64.whl
+          echo "WHEEL: ${WHEEL}"
+          echo "WHEEL=${WHEEL}" >> $GITHUB_ENV
```

Comment:
Ugh. If version is `0.6.0-a.2` then the actual wheel is `dist/kaskada-0.6.0a2-cp38-abi3-manylinux_2_28_x86_64.whl`.

So we need to do some post-processing on the pre-release version. This is going to get verbose.